bots/sg_single_rule_delete.py

```diff
@@ -1,362 +0,0 @@
-'''
```

Comment:
Why you deleted it?

Reply:
I will create new pull request only for this change

Reply:
?

Reply:
I hope now it's fixed after my commit

Reply:
I Reverted the delete